scw_serverless/gateway/serverless_gateway.py

```diff
@@ -0,0 +1,50 @@
+import shutil
+import subprocess
+
+import click
+
+from scw_serverless.config.route import GatewayRoute
+
+GATEWAY_CLI = "scwgw"
+
+
+class ServerlessGateway:
+    """Manage routes on a Kong Gateway with scwgw."""
+
+    cli: str
+
+    def __init__(self) -> None:
+        cli = shutil.which(GATEWAY_CLI)
+
+        if not cli:
+            click.secho(
+                "scwgw was not found in $PATH, "
+                + "you can install scwgw by running: pip install scwgw",
+                fg="red",
+            )
+            raise RuntimeError("scwgw is not installed")
+
+        self.cli = cli
```

Comment:
Initially, I wanted to run `scwgw check` here to fail fast if the gateway container were not ready, but the problem was that env vars are not properly passed to the subprocess (intended behavior to avoid security issues) and that output parsing was a bit ugly.

What do you think?